.env.example

```diff
@@ -7,6 +7,10 @@ REQUEST_SPREADSHEET_ID=the_last_part_of_the_url
 REQUEST_SHEET_DATA_RANGE='Request Private Beta'!A1
 REQUEST_SHEET_HEADER_RANGE='Request Private Beta'!A1:Y1
 
+MAILING_LIST_SPREADSHEET_ID=the_last_part_of_the_url
+MAILING_LIST_SHEET_DATA_RANGE=Register!A1
```

Comment:
We should update these to be the actual values of the ranges we use.  It will make it easier for anyone else trying to get things going.